portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -28,17 +28,16 @@
 import java.time.format.DateTimeFormatter;
 import java.time.LocalDateTime; 
 
-/** Servlet that returns a comment*/
+/* Servlet that creates a comment*/
 @WebServlet("/new-comment")
 public class NewCommentServlet extends HttpServlet {
   
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("commentText");
     long timestamp_millis = System.currentTimeMillis();
-    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM/dd/yyyy");
-    LocalDateTime now = LocalDateTime.now();
-    String currentDate = formatter.format(now).toString();
+    DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy/MM/dd");
+    String currentDate = formatter.format(LocalDateTime.now()).toString();
```

Comment:
Best to defer casts like this until you need them. Avoid dealing with "untyped" strings until you absolutely have to.

(i'm suggesting you make this a DateTime, then call .toString() at the usage site.

Reply:
Can you elaborate on "untyped" strings? I haven't heard of that before. 
commentDateFormat.format(LocalDateTime.now()) gives me a string so does removing .toString() suffice?

Reply:
Sorry, took liberties with my wording. This was really a "Nit", but I didn't note it at the time.

Strings containing data are more opaque to readers of your code (particularly future readers, including yourself) than are properly typed values. In this case the properly typed value here is DateTime. This applies doubly to time and date like values because there are a huge number of possible representations of them.

Hence my original comment: It's best to defer conversion from self evident data types into opaque strings until you absolutely have to. This is a lesson a lot of people learn the hard way when their system passes a lot of "strings" containing data around and becomes hard to understand.

Reply:
I just looked at the API here, and I'm a bit confused. It looks like formatter.format(...) is already returning a string. In that case, what is toString() doing here?